core/src/main/java/dev/hypera/scaffolding/Scaffolding.java

```diff
@@ -49,14 +46,14 @@ public final class Scaffolding {
     private static final @NotNull NBTSchematicReader MC_EDIT_SCHEMATIC_READER = new MCEditSchematicReader();
     private static final @NotNull NBTSchematicReader SPONGE_SCHEMATIC_READER = new SpongeSchematicReader();
 
-    private Scaffolding() {}
+    private Scaffolding() {
+    }
```

Comment:
There should be an empty line above this line.

Reply:
Already is?